gobblin-modules/gobblin-http/src/main/java/gobblin/writer/AvroHttpWriterBuilder.java

```diff
@@ -36,4 +38,8 @@ public AvroHttpWriterBuilder fromConfig(Config config) {
     this.responseHandler = new HttpResponseHandler();
     return this;
   }
+
+  public String getLimiterKey (State state) throws NotConfiguredException {
+    return AvroHttpWriterBuilder.class.getName();
```

Comment:
Shouldn't it be `url.getAuthority`? the path doesn't mean much, same path in different endpoints don't produce more load, whereas different paths in the same endpoint may.